source/community/release_notes/sumac/customizing_header.rst

```diff
@@ -0,0 +1,162 @@
+Customizing Site Header Using Frontend "Slots"
+##############################################
+
+Utilizing *frontend plugin slots*, site operators now have the ability to
+customize various portions of the site header.
+
+A “frontend plugin slot” refers to an area of a web page - comprising one or
+more visual elements - that can be “swapped out” with other visual elements. For
+example, one new plugin slot allows you to remove the "Help" button in the header.
+
+Overhead and hassle is minimized using the plugin slot system. Site operators
+can leverage a plugin slot using a configuration file; the codebase does not
+need to be copied (“forked”) nor are extensive changes needed. A snippet of code
+is all that is needed to use a plugin slot. A site operator places that code
+within a configuration file. Site operators should refer to the src/plugin-slots
+directory within each MFE's codebase to view documentation for that MFE's plugin
+slot(s).
+
+In these release notes, we'll cover three example use cases for the header. Full
+documentation is available `within the frontend-component-header codebase
+<https://github.com/openedx/frontend-component-header/tree/master/src/plugin-slots>`_.
+
+.. contents::
+  :local:
+  :depth: 1
+
+Simple example: Removing help button
+************************************
+
+In the default learner dashboard header, the content contains a "Help" button
+to the left of the logged in user's username:
+
+   .. image:: /_images/release_notes/sumac/sumac-header-easy-before.png
+
+With frontend plugin slots, you can now hide the "Help" button:
+
+   .. image:: /_images/release_notes/sumac/sumac-header-easy-after.png
+
+For site operators, here is a sample of the ``env.config.jsx`` file used to hide
+the the content:
+
+.. toggle::
+
+    .. code-block:: javascript
+
+        import { PLUGIN_OPERATIONS } from '@openedx/frontend-plugin-framework';
+
+        const config = {
+          pluginSlots: {
+            learning_help_slot: {
+              plugins: [
+                {
+                  op: PLUGIN_OPERATIONS.Hide,
+                },
+              ]
+            },
+          },
+        }
+
+        export default config;
+
+Medium-effort example: Replacing "Course Info" with a custom component
+**********************************************************************
+
+In the default learner dashboard header, the content contains information about
+the current course's organization and title in the top left corner:
+
+   .. image:: /_images/release_notes/sumac/sumac-header-medium-before.png
+
+With frontend plugin slots, you can now create your own custom component (in
+this case, an icon of a book) in place of the course info component:
+
+   .. image:: /_images/release_notes/sumac/sumac-header-medium-after.png
+
+For site operators, here is a sample of the ``env.config.jsx`` file used to
+render the custom component:
+
+.. toggle::
+
+    .. code-block::
```

Comment:
@brian-smith-tcril - if I define this as a js block I get a lexical error, but no detail as to what was wrong (did some googling, there's no way to get deeper insight). Could you check if I transcribed this properly, or if some syntax/spacing is off?

Reply:
does rst support `.jsx` code blocks? `.jsx` code uses extensions not available in vanilla `.js` (specifically the ability to just throw HTML in there directly)

Reply:
It looks like it should support it https://pygments.org/docs/lexers/#lexers-for-jsx-react-and-tsx-typescript-flavor using the shortname "jsx"

Reply:
Yup. Thanks for checking that, I'm not sure why that didn't click for me.